docs/_static/headless/openapi-specification/openapi.yaml

```diff
@@ -0,0 +1,2442 @@
+        - "Account: Email"
+      summary: Change primary email address
+      description: |
+        Used to change primary email address to a different one. Note that only verified email addresses
+        can be marked as primary.
+      parameters:
+        - $ref: "#/components/parameters/Client"
+        - $ref: "#/components/parameters/SessionToken"
+      requestBody:
+        $ref: "#/components/requestBodies/MarkPrimaryEmail"
+      responses:
+        "200":
+          $ref: "#/components/responses/EmailAddresses"
+        "400":
+          description: An input error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+              examples:
+                invalid_code:
+                  $ref: "#/components/examples/InvalidEmail"
+    delete:
+      tags:
+        - "Account: Email"
+      summary: Remove an email address
+      description: |
+        Used to remove an email address.
+      parameters:
+        - $ref: "#/components/parameters/Client"
+        - $ref: "#/components/parameters/SessionToken"
+      requestBody:
+        $ref: "#/components/requestBodies/Email"
+      responses:
+        "200":
+          $ref: "#/components/responses/EmailAddresses"
+        "400":
+          description: An input error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+              examples:
+                invalid_code:
+                  $ref: "#/components/examples/InvalidEmail"
+  ######################################################################
+  # Account: 2FA
+  ######################################################################
+  /_allauth/{client}/v1/account/authenticators:
+    get:
+      tags:
+        - "Account: 2FA"
+      summary: List authenticators
+      parameters:
+        - $ref: "#/components/parameters/Client"
+        - $ref: "#/components/parameters/SessionToken"
+      responses:
+        "200":
+          $ref: "#/components/responses/Authenticators"
+        "401":
+          $ref: "#/components/responses/Authentication"
+        "410":
+          $ref: "#/components/responses/SessionGone"
+  /_allauth/{client}/v1/account/authenticators/totp:
+    get:
+      tags:
+        - "Account: 2FA"
+      summary: TOTP authenticator status
+      description: |
+        Retrieve the information about the current TOTP authenticator, if any.
+      parameters:
+        - $ref: "#/components/parameters/Client"
+        - $ref: "#/components/parameters/SessionToken"
+      responses:
+        "404":
+          $ref: "#/components/responses/TOTPAuthenticatorNotFound"
+        "200":
+          $ref: "#/components/responses/TOTPAuthenticator"
+    post:
+      tags:
+        - "Account: 2FA"
+      summary: Activate TOTP
+      description: |
+        Activate TOTP authentication by means of inputting a code. The code
```

Comment:
I feel like this could be rephrased as something like:

> The code should be provided from the consuming TOTP authenticator application which was generated using the TOTP authenticator secret retrieved from the TOTP authenticator status endpoint.

Reply:
Applied